vars/runIntegTestScript.groovy

```diff
@@ -4,11 +4,28 @@ void call(Map args = [:]) {
     def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: args.buildManifest))
     String artifactRootUrl = buildManifest.getArtifactRootUrl(jobName, args.buildId)
     echo "Artifact root URL: ${artifactRootUrl}"
-    
+
+    String paths = generatePaths(buildManifest, artifactRootUrl)
+    echo "Paths: ${paths}"
+
     sh([
         './test.sh',
         'integ-test',
         "${args.testManifest}",
-        "--paths opensearch=${artifactRootUrl}",
+        "--paths ${paths}",
     ].join(' '))
+}
+
+String generatePaths(buildManifest, artifactRootUrl) {
+    String name = buildManifest.build.name
+    String version = buildManifest.build.version
+    String platform = buildManifest.build.platform
+    String architecture = buildManifest.build.architecture
+
+    // TODO: Needs to be latest
+    String tempBuildId = "740"
```

Comment:
When should this change be done? Can you create an issue if not now so that we don't miss it?

Reply:
If this PR gets merged in before this https://github.com/opensearch-project/opensearch-build/pull/1569, I can add a comment to that PR to update that piece in the PR.

Reply:
Addressed!